sw.js

```diff
@@ -0,0 +1,84 @@
+// I am a service worker for the remote newtab page.
+
+// NOTE: To invalidate cached resources, change the version number below.
+const VERSION = '1';
+const CACHE_NAME = 'remote-newtab-v' + VERSION;
+
+self.addEventListener('install', function (event) {
+  console.log('I am installed.');
+
+  // NOTE: If we will always serve from a static path (as opposed to a dynamic
+  // path like /%VERSION%/%CHANNEL%/%LOCALE%/), we can do this below
+  // instead of the more involved fetch handler below.
```

Comment:
I was under the impression that the full path needed to be specified here. If we can do paths like `./` instead, then this is more feasible.